addons/chatmon/readme.md

```diff
@@ -21,28 +21,62 @@
 #### Tigger Fields:
 * `from`: a set of chat name you would like the trigger to happen on, the full list of chat modes can be found here [resources/chat.lua](https://github.com/Windower/Resources/blob/master/resources_data/chat.lua)
```

Comment:
I would adjust this to not mention "chat modes", since it may give people the impression that they need modes, not just names.